schema.sql

```diff
@@ -1,5 +1,12 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    /*id INT SERIAL PRIMARY KEY*/
+    id INT GENERATED BY DEFAULT AS IDENTITY,
+    name VARCHAR(255),
+    date_of_birth DATE,
+    escape_attempts INT,
+    neutered BOOLEAN,
```

Comment:
**[Optional]**

- [ ] It would be nice if you would pass true or false values to neutered column to use this column correctly because it has a datatype boolean :thinking: